src/src/EmailBundle/Command/SendDailyNotificationCommand.php

```diff
@@ -45,7 +46,15 @@ private function sendNotificationForEvent($event, $controller)
         $context = $this->getContextForNotificationEmail($event, $controller);
         foreach ($event->getOrganizers() as $eventOrganizer) {
             $organizer = $eventOrganizer->getOrganizer();
+            $em = $this->getContainer()->get('doctrine')->getManager();
+            $notConfirmedRegistrations = $em->getRepository(Registration::class)->findBy(
+                ['event'=> $event, 'confirmed' => null]
+            );
+            $allRegistrations = $em->getRepository(Registration::class)->findBy(
+                ['event'=> $event]
+            );
             $context['organizer'] = $organizer;
+            $context['confirmed_registrations_count'] = count($allRegistrations) - count($notConfirmedRegistrations);
```

Comment:
`count($allRegistrations)` nacita a preiteruje vsetky data v `php`cku.

Ak sa ti chce tak to zmen https://stackoverflow.com/a/9215880/2317085